opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/TracerAutoConfiguration.java

```diff
@@ -15,6 +17,8 @@
  * @author Pavol Loffay
  */
 @Configuration
+@EnableConfigurationProperties(WebTracingProperties.class)
+@ConditionalOnProperty(name = "opentracing.spring.web.enabled", havingValue = "true", matchIfMissing = true)
```

Comment:
I don't think it's needed here. This configuration provides tracer bean for different instrumentations and auto-configs